app/namespaces/data_provider/data_provider_schema.py

```diff
@@ -32,8 +32,11 @@ class RecordsSchema(Schema):
                                                                  "estimate_grade", "isotypes_reported", "created_at",
                                                                  "pin_longitude", "pin_latitude", "pin_region_type"])))
     # Date fields should be supplied as unix timestamp
-    start_date = fields.String()
-    end_date = fields.String()
+    sampling_start_date = fields.String()
```

Comment:
this name change might break the front end so we need to coordinate pushes

Reply:
I do agree with the name change tho

Reply:
@brett-stephen